docs/modules/ROOT/pages/references/architecture/control-api-design.adoc

```diff
@@ -0,0 +1,181 @@
+= APPUiO Control API Design
+
+This page describes the overall APPUiO Control API ("API" in the rest of this page) design.
+It adheres to the decision taken in xref:explanation/decisions/control-api.adoc[APPUiO Control API].
+The design follows the features asked in xref:explanation/system/details-ui.adoc[System idea: Graphical user interface].
+
+image:reference/control-api-design.drawio.svg[]
+
```

Comment:
I actually don't understand what this diagram is trying to visualize :see_no_evil:  but let's discuss that separately.